resources/views/travel/index.blade.php

```diff
@@ -0,0 +1,72 @@
+@extends('layouts/app')
+
+@section('title')
+Travel | {{ config('app.name') }}
+@endsection
+
+@section('content')
+    @component('layouts/title')
+        Travel
+    @endcomponent
+
+    <div class="row">
+        <div class="col-md-12">
+            <p>You are assigned to the following upcoming travel.</p>
+
+            <table class="table table-sm">
+              <tbody>
+                <tr>
+                  <th scope="row">Name</th>
+                  <td>{{ $travel->name }}</td>
+                </tr>
+                <tr>
+                  <th scope="row">Destination</th>
+                  <td>{{ $travel->destination }}</td>
+                </tr>
+                <tr>
+                  <th scope="row">Primary Contact</th>
+                  <td>{{ $travel->primaryContact->full_name }}</td>
+                </tr>
+                <tr>
+                  <th scope="row">Departure Date</th>
+                  <td>{{ $travel->departure_date->format("l, F j, Y") }}</td>
+                </tr>
+                <tr>
+                  <th scope="row">Return Date</th>
+                  <td>{{ $travel->return_date->format("l, F j, Y") }}</td>
+                </tr>
+                <tr>
+                  <th scope="row">Fee</th>
+                  <td>${{ $travel->fee_amount }}</td>
+                </tr>
+                <tr>
+                  <th scope="row">Costs Included</th>
+                  <td>@markdown($travel->included_with_fee)</td>
+                </tr>
+                @if($travel->not_included_with_fee)
+                <tr>
+                  <th scope="row">Not Included</th>
+                  <td>@markdown($travel->not_included_with_fee)</td>
```

Comment:
Is this injection-safe?

Reply:
I think we looked at this for membership agreements. You can run arbitrary code in browser but not on the server.

It's limited to PM+ so I think it should be fine.